dasharo-compatibility/usb-hid-and-msc-support.robot

```diff
@@ -66,11 +66,11 @@ USB002.001 USB keyboard detected in FW
     ...    correctly by the firmware and all basic keys work
     ...    according to their labels.
     Power On
-    ${setup_menu}=    Enter Setup Menu Tianocore And Return Construction
-    ${device_mgr_menu}=    Enter Submenu From Snapshot And Return Construction
-    ...    ${setup_menu}
-    ...    Device Manager
-    Save Changes And Reset
+    Enter UEFI Shell
+    Set Prompt For Terminal    Shell>
+    Execute UEFI Shell Command    devices
+    ${out}=    Read From Terminal Until Prompt
```

Comment:
```suggestion
    ${out}=    Execute UEFI Shell Command    devices
```

`Execute UEFI Shell Command` executes command and returns it's output. Current implementation also doesn't use prompt but calculated `Sleep` time before reading everything available on serial.

Reply:
Done  63e9feaf935b928e2702b03512430d964706fcfb